src/adapter/src/catalog/builtin_table_updates.rs

```diff
@@ -1162,4 +1164,26 @@ impl CatalogState {
             diff,
         }
     }
+
+    pub fn pack_session_update(
+        &self,
+        id: ConnectionId,
+        role_id: RoleId,
+        diff: Diff,
+    ) -> BuiltinTableUpdate {
+        // We use the system clock to determine when a session
+        // connected to Materialize. This is not intended to be 100%
+        // accurate and correct, so we don't burden the timestamp
+        // oracle with generating a more correct timestamp.
```

Comment:
Note: This is the same approach we take for subscription start times.

Reply:
Makes sense to me!